resources/features/Sic.feature

```diff
@@ -0,0 +1,243 @@
+Feature: The top down classification method considers units based on their SIC section letter (subdivision) then works down 2, 3, 4 and 5 digit
+  level until a clear classification is determined.
+
+  The method uses the following substrings of sic07:
+  first 2 characters as division,
+  first 3 characters as group,
+  first 4 characters as class,
+  and the whole whole SIC.
+
+  It also uses subdivision which is determined by the division and the table can be viewed in confluence.
+  https://collaborate2.ons.gov.uk/confluence/pages/viewpage.action?pageId=5386186
+
+  The method works by drilling down these levels until it has a single sic07 for each enterprise which is output.
+
+  There are special cases for divisions 46 and 47 which can also be viewed at the above confluence page.
+
+
+  @Happy Path
+
+  Scenario Outline: Happy Path - Original Happy Path
+    Given input:
+      | ern | lurn | sic07 | employees |
+      | 123 | 1    | 63111 | 504       |
+      | 123 | 4    | 41202 | 844       |
+      | 123 | 6    | 69201 | 188       |
+      | 123 | 10   | 46444 | 301       |
+      | 123 | 13   | 47344 | 300       |
+      | 123 | 14   | 47844 | 900       |
+      | 123 | 15   | 47944 | 950       |
+      | 345 | 2    | 63110 | 46        |
+      | 345 | 7    | 12312 | 100       |
+      | 345 | 11   | 47544 | 500       |
+      | 345 | 9    | 46212 | 300       |
+      | 456 | 8    | 46123 | 500       |
+      | 456 | 3    | 63120 | 540       |
+      | 654 | 5    | 78109 | 517       |
+      | 654 | 12   | 47144 | 1000      |
+    When the Sic method is calculated
+    Then the Sic results table is produced:
+      | ern | sic07 |
+      | 345 | 47544 |
+      | 456 | 63120 |
+      | 654 | 47144 |
+      | 123 | 47944 |
+
+  @JVM
+    Examples:
+      | language |
+      | Scala    |
+
+  @Happy Path
+
+  Scenario Outline: Happy Path - A Results table is which hits a few endpoints of the decision tree
```

Comment:
The 'Original Happy Path' and this scenario are not both required, they don't add any extra value.
As this scenario represents lurn in the correct format I would drop the 'original happy path' from the feature.